snapcraft/internal/errors.py

```diff
@@ -224,6 +243,7 @@ def __init__(self, *, part_name, other_part_name, conflict_files):
 class MissingCommandError(SnapcraftError):
 
     fmt = (
+        'Failed to run command: '
         'One or more required commands are missing, please install:'
```

Comment:
This doesn't read particularly well: "Failed to run command: One or more required commands[...]"

What about "Failed to run command: One or more requires packages[...]" or "utilities" or something similar?

Reply:
I think the problem here is that technically speaking this does refer to the command, and not the package. If we wanted to be more helpful, we'd have to move check_for_command into the Repo class and have it lookup the corresponding package.

Reply:
I'm leaving it as "packages" for now, but this message seems to require more work